.travis.yml

```diff
@@ -112,18 +112,22 @@ matrix:
       script:
         - "./build/travis/job_macos/script.sh"
 
-
 notifications:
-  recipients:
-    - $NOTIFICATION_EMAIL
   email:
+    recipients:
+      - "build@musescore.org"
     on_success: change
     on_failure: always
   irc:
     channels:
-      - $NOTIFICATION_IRC
+      - "chat.freenode.net#musescore"
     on_success: change
     on_failure: change
+    template: # https://docs.travis-ci.com/user/notifications/#IRC-notification
```

Comment:
Sure, let's add skip_join: true
The message is one click away, so we don' really need it. It's a notification after all, shorter is better. I would happily keep the default message. It also has the advantage to be default and so similar in other projects, and so easier to parse visually.